training/deepspeech_training/util/sample_collections.py

```diff
@@ -570,13 +599,19 @@ def samples_from_sources(sample_sources, buffering=BUFFER_SIZE, labeled=None, re
 
     Returns
     -------
-    iterable of util.sample_collections.LabeledSample (labeled=True) or util.audio.Sample (labeled=False) supporting len
+    iterable of util.sample_collections.PackedSample if a single collection is provided, wrapping
+        LabeledSample (labeled=True) or util.audio.Sample (labeled=False) supporting len
+    or LabeledSample / util.audio.Sample directly, if multiple collections are provided
     """
     sample_sources = list(sample_sources)
     if len(sample_sources) == 0:
         raise ValueError('No files')
     if len(sample_sources) == 1:
         return samples_from_source(sample_sources[0], buffering=buffering, labeled=labeled, reverse=reverse)
-    cols = [samples_from_source(source, buffering=buffering, labeled=labeled, reverse=reverse)
+
+    # Otherwise, if we wish to interleave based on duration, we have to unpack the audio (on the fly)
```

Comment:
I don't understand this comment, in particular starting with "otherwise". We're always unpacking here, right?

Reply:
The "otherwise" was just referring to the if statements above. "if the length is 0, blah. if the length is 1, blah. otherwise, we have to unpack". I might have a bad habit of reading python as english. Fixed.